advanced/button-interrupt/exercise/Cargo.toml

```diff
@@ -17,7 +17,7 @@ default = ["native"]
 native = ["esp-idf-sys/native"]
 
 [dependencies]
-esp-idf-sys = { version = "0.30", features = ["binstart"] }
+esp-idf-sys = { version = "=0.31.5", features = ["binstart"] }
```

Comment:
There are few version bumps that update esp related dependencies. Would you mind putting that into a separate Pull Request? If that is a required change to make the gitpod environment work, please add a comment.

Once PR #66 is in `main`, we can use the CI pipeline to ensure no breaking change is introduced when updating dependencies.

Reply:
Yes, those are addressed in a different PR. Sorry, maybe I should have closed this PR until https://github.com/ferrous-systems/espressif-trainings/pull/69 gets merged to avoid confussion.